.github/workflows/plugins-benchmark-pr.yml

```diff
@@ -103,7 +103,7 @@ jobs:
       pull-requests: write
     steps:
       - name: Comment PR
-        uses: thollander/actions-comment-pull-request@v3
+        uses: thollander/actions-comment-pull-request@65f9e5c9a1f2cd378bd74b2e057c9736982a8e74 # v3
```

Comment:
In my opinion, this one is good, but we don't need to use hashes on the GitHub owned actions.

Reply:
I did consider not using hashes for GitHub's official actions as they should (hopefully) be more secure than third-party actions, but official actions aren't immune from being compromised, so pinned them just to help me sleep better at night. 😄

Reply:
Not immune. But highly unlikely. I think the maintenance cost is too high to use hashes on GH maintained actions.

Reply:
Dependabot is active in this repo so it'll sort that for us.